integreat_cms/cms/fixtures/test_data.json

```diff
@@ -355,6 +355,23 @@
       "message_partial_live_content_not_available": "Ein Teil von der Seite ist nicht in Ihrer Sprache. Der Teil ist aber in diesen Sprachen verfügbar:"
     }
   },
+  {
+    "model": "cms.language",
+    "pk": 7,
+    "fields": {
+      "slug": "hidden",
+      "bcp47_tag": "hid",
+      "native_name": "Hidden language",
+      "english_name": "Hidden language",
+      "text_direction": "LEFT_TO_RIGHT",
+      "primary_country_code": "hi",
+      "created_date": "2022-11-30T09:16:00.000Z",
+      "last_updated": "2022-11-30T09:16:00.000Z",
+      "table_of_contents": "Table of Contents",
+      "message_content_not_available": "This Language may exist but is marked as hidden. We test the behavior when it is marked as hidden:",
+      "message_partial_live_content_not_available": "This Language may exist but is marked as hidden."
```

Comment:
Also, I'd use the correct fallback text here. The text above does not really make sense...